build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-android-compat.gradle.kts

```diff
@@ -9,7 +9,7 @@ pluginManager.withPlugin("org.gradle.java-base") {
     apply(plugin = "ru.vyarus.animalsniffer")
 
     configure<AnimalSnifferExtension> {
-        sourceSets = listOf((project.extensions.getByName("sourceSets") as SourceSetContainer).getByName("main"))
+        defaultTargets = setOf("jvmMain")
```

Comment:
based on changes in https://github.com/Kotlin/kotlinx.coroutines/commit/6baf7c821ed9d7ee5db8b6353c24bcf614fc1826#diff-bfdaf463516db2c76758e3e9823610f7ba309cc3dbd20592ce880facf20e7dd0R306-R308